Google.Api.Generator/Generation/SnippetCodeGenerator.cs

```diff
@@ -169,7 +172,30 @@ public NamespaceDeclarationSyntax StandaloneSnippet(SourceFileContext ctx)
                     }
                     ns = ns.AddMembers(cls);
                 }
-                return ns;
+
+                return WithRegionTag(ns);
+            }
+
+            private NamespaceDeclarationSyntax WithRegionTag(NamespaceDeclarationSyntax ns)
+            {
+                string effectiveShortName = TargetMethod.Svc.DefaultHostServiceName ?? "unknown";
+                string effectiveVersion = TargetMethod.Svc.PackageVersion is null ? "" : $"_{TargetMethod.Svc.PackageVersion}";
+                string syncText = Sync ? "sync" : "async";
+                string effectiveDisambiguation = RegionTagDisambiguation is null ? "" : $"_{RegionTagDisambiguation}";
+
+                // {apishortname}_{apiVersion}_generated_{serviceName}_{rpcName}_{sync|async}_{disambiguation}
+                string regionTagName =
+                    $"{effectiveShortName}" +
+                    $"{effectiveVersion}" +
```

Comment:
Aha... I'd been wondering why we actually need the version...

Reply:
Yep :)